pkg/command/plugin.go

```diff
@@ -161,7 +161,7 @@ func newDescribePluginCmd() *cobra.Command {
 			}
 			pluginName := args[0]
 
-			if !configtypes.IsValidTarget(targetStr) {
+			if !configtypes.IsValidTarget(targetStr, true) {
 				return errors.New("invalid target specified. Please specify correct value of `--target` or `-t` flag from 'kubernetes/k8s/mission-control/tmc'")
```

Comment:
I see you haven't updated these error message.  I assume it is because a user should never have to specify the "global" target?  The only time we could have a target conflict is between "tmc" and "k8s", since we agree that if a plugin is for the "global" target, it can't be for any other target.

However, it is possible for someone to write a plugin for "k8s" and another for "global" and use the same name, and then install them using `--local` (or with different plugin sources).  I suggest we open a separate issue as there are other protections we should probably put in place for that case: right now I believe we'll try to install both.

Reply:
I was thinking we can fix this as a separate fix.

I was going to ping you for this one as well. I remember we discussed a possible issue where a plugin is a `global` plugin and the same-named plugin is available under the `tmc` target but was not possible to install.

Have we filled an issue to track that? I was planning to fix this as part of the separate bug fix that we discovered earlier.

Reply:
Filled: https://github.com/vmware-tanzu/tanzu-cli/issues/77